src/EventStore.Core/Index/IndexMap.cs

```diff
@@ -394,7 +415,40 @@ public void SaveToFile(string filename)
             }
         }
 
+		//TODO: this shouldn't really be called manual table, come up with a better name
```

Comment:
Do we need to resolve this before merge?

Reply:
no, i changed the behavior so the comment became irrelevant, know it does get a table for manual merge. will delete